core/frontend/src/components/app/SettingsMenu.vue

```diff
@@ -72,9 +90,15 @@ export default Vue.extend({
       show_dialog: false,
       show_reset_dialog: false,
       settings,
+      pirate_warning: false,
     }
   },
   methods: {
+    togglePirateWarning(): void {
+      if (settings.is_pirate_mode) {
+        this.pirate_warning = true
```

Comment:
`this.pirate_warning = settings.is_pirate_mode`, otherwise it'll never the false.

Reply:
Don't know why I wrote it this way 😅